app/src/Model/Table/ArticlesTable.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+// src/Model/Table/ArticlesTable.php
```

Comment:
I've 1 question, is it OK to write the comment here (and occur in other lines of code too)?

Reply:
Since it's copied from the CakePHP tutorial, it's OK for now. No need to remove this line and others too, but I hope it's not practice in the actual project.

Reply:
is not okay those comments, so they were removed